src/com/lal/fiboapp/FibonnaciApp.java

```diff
@@ -1,3 +1,4 @@
+package com.lal.fiboapp;
```

Comment:
Inconsistency in repository name and package name. It would be better if repository name and package name is changed to fibonnaciApp which is same as the application name

Reply:
I don't have the access to change the repository name. I can only change after changing the repository name. I would ask santosh to make the changes.

Reply:
good catch @Laldhan-glitch . I am going to change it right away.